app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= link_to user_path(user) do %>
+  <div class="user-container">
+    <div class='image-container'>
+      <img src="<%= user.photo %>" alt="<%= user.name %>" />
+    </div>
+    <div class='text-container'>
+      <h2><%= user.name %></h2>
+      <p>Number of posts: <%= user.posts_counter %></p>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] As required for the project, kindly ensure that your design follows the given **[wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png),** below is how looks a user id card. To fix that you need to check the `width` and the `height` of the user photo as this issue appears in all views with user's photo 
![image](https://github.com/newhoteng/Blog-app/assets/83042742/6328dd5e-6afb-4294-a51c-d414fc673717)